sos/report/plugins/rhc.py

```diff
@@ -46,4 +47,19 @@ def postproc(self):
         self.do_path_regex_sub("/etc/rhc/workers/foreman_rh_cloud.toml",
                                r"(FORWARDER_PASSWORD\s*=\s*)(.+)(\"\,)",
                                r"\1********\3")
+
+        # hide ssh host keys from rhc-worker ansible playbooks
+        # Example for scrubbing one of the ssh keys
+        #
+        # "ansible_ssh_host_key_ecdsa_public": "ABCDEFGHIJ",
+        #
+        # to
+        #
+        # "ansible_ssh_host_key_ecdsa_public": ********,
+
+        path = "/var/log/rhc-worker-playbook/ansible/*"
+        regexp = r"(\s*\"ansible_ssh_host_key_)(.+)(_public\":\s*)(.+)(\,)"
```

Comment:
There is a tiny probability some SSH public key will be the last item in the JSON map, hence without trailing comma. So `(\,)` should be `(\,|$)` I guess..?